openvpn/torguard/Australia.LOC.1.ovpn

```diff
@@ -45,6 +26,13 @@ c183f65da5d3c9001a8cbdb96c793d93
 38d938bd0597c426b0e440434f2c451f
 -----END OpenVPN Static key V1-----
 </tls-auth>
+resolv-retry infinite
+nobind
+cipher AES-128-CBC
+auth-user-pass
```

Comment:
this needs fixing,
needs tobe
 auth-user pass /config/openvpn-credentials.txt

Reply:
I've been thinking about it - it works already as it is, because by default we're automatically replacing `auth-user-pass` with `auth-user pass /config/openvpn-credentials.txt` in [here](https://github.com/haugene/docker-transmission-openvpn/blob/master/openvpn/modify-openvpn-config.sh#L23).

Is it scalable to require all contributors to change it and to verify that everyone has adjusted it accordingly if we're automatically going to set it correctly in the container anyway?

Adding and/or updating configs becomes much easier if you can leave most of the configs as provided by a VPN provider, considering that the container automatically adjusts what needs adjusting.

Reply:
My bad, I forgot that we do this check already :) it used to be that everyone had to update the configuration but no longer I think